index.html

```diff
@@ -72,20 +74,23 @@ <h2>Tonic</h2>
                     <button class="btn-action">See project</button>
                 </div>
             </div>
-            <div class="work-container">
+            <div class="work-container" id="reverse">
                 <div class="card">
-                    <img src="images/Snapshoot Portfolio (5).svg" alt="">
+                    <img src="images/desktop2.png" alt="">
                 </div>
                 <div class="left-block">
                     <h2>Multi-Post Stories</h2>
                     <ul class="position">
-                        <li>CANOPY<i class="fa fa-circle"></i></li>
-                        <li>Back End Dev<i class="fa fa-circle"></i></li>
+                        <li>FACEBOOK <i class="fa fa-circle"></i></li>
+                        <li>Full Stack Dev <i class="fa fa-circle"></i></li>
                         <li>2015</li>
                     </ul>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <p>Experimental content creation feature that allows users to
+                        add to an exiting story over the course of a day without
+                        spamming their friends</p>
```

Comment:
Your design
![image](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/78251927/0a585980-8c02-4ce1-a082-58b7ac7a2bb9)

After creating the `Desktop version`, kindly ensure that the mobile design still looks like expected while switching from desktop to mobile. 🤗👍